### ISSUE TITLE
NPE error loading campain

### ISSUE THREAD

**Issue by bczretribution** [MegaMek/mekhq#592]
### Environment
MekHQ 0.43.6
windows 8.1
Java 1.8.0_121

### Description
Two of my related campaign files from a recent save has developed a NPE when loading.

[FailToLoad.zip](https://github.com/MegaMek/mekhq/files/1601646/FailToLoad.zip)

**Comment by HammerGS** [comment]
From this forum post - https://www.megamek.org/bb/thread-2489.html

NPE from File
```
java.lang.NullPointerException at mekhq.campaign.universe.RandomFactionGenerator.updateBorders(RandomFactionGenerator.java:495)
at mekhq.campaign.universe.RandomFactionGenerator.updateTables(RandomFactionGenerator.java:454)
at mekhq.campaign.Campaign.restore(Campaign.java:3110)
at mekhq.gui.dialog.DataLoadingDialog$Task.doInBackground(DataLoadingDialog.java:195)
at mekhq.gui.dialog.DataLoadingDialog$Task.doInBackground(DataLoadingDialog.java:117)
at javax.swing.SwingWorker$1.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at javax.swing.SwingWorker.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)

```

**Comment by NickAragua** [comment]
Fixed with PR #593
